lib/swig/customParamValues.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CUSTOMPARAMVALUES_H
+#define CUSTOMPARAMVALUES_H
+namespace CustomValues
+{
+class ParamValues
+{
+public:
+    template<typename T>
```

Comment:
How come this is templated?

Reply:
Well, it is just a `Snippet::ValueBase` with a relaxed constructor... And yes, the template here is absolutely unnecessary :sweat_smile:

Reply:
Hahah